db/dataset_eval.py

```diff
@@ -129,40 +143,29 @@ def validate_dataset_contents(dataset):
 
 
 def get_next_pending_job():
-    # TODO: This should return the same data as `get_job`, so
-    #       we run 2 queries, however it would be more efficient
-    #       to do it in 1 query
+
     with db.engine.connect() as connection:
         query = text(
-            """SELECT id::text
+            """SELECT %s
                  FROM dataset_eval_jobs
-                WHERE status = :status
-                  AND eval_location = 'local'
-             ORDER BY created ASC
-                LIMIT 1""")
+                 JOIN dataset_snapshot ON dataset_snapshot.id = dataset_eval_jobs.snapshot_id
```

Comment:
In general, we try to keep our SQL keywords right aligned.

```
  SELECT %s
    FROM dataset_eval_jobs
    JOIN dataset_snapshot
      ON dataset_snapshot.id = dataset_eval_jobs.snapshot_id
   WHERE status = :status
     AND eval_location = 'local'
ORDER BY created ASC
   LIMIT 1
```

Reply:
https://github.com/metabrainz/guidelines/blob/master/Python.md#sql

Reply:
often while adding adding stuff to the query i loose track of the alignment :), i'll take care of it from now on